src/utils/axios.js

```
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const api = axios.create({baseURL: 'https://jacks-api.onrender.com'})
+
+export async function getArticles() {
+    return api.get('/api/articles')
+    .then((response) => {
+        return response.data.articles
+    })
+    .catch((error) => {
+
+    })
+}
+
+export async function getArticleById(article_id) {
+    return api.get(`/api/articles/${article_id}`)
+    .then((response) => {
+        return response.data.article
+    })
+    .catch((error) => {
+
+    })
+}
```

I would avoid having empty catch blocks here - the errors that enter them will go under the radar. Also, you're best to catch any errors in a place where you intend to do something with them - consider relocating to the components themselves